docs/reference/tab-widgets/quick-start-install.asciidoc

```diff
@@ -21,9 +21,37 @@ Desktop].
 ----
 docker network create elastic
 docker pull {docker-repo}:{version}
-docker run --name es01-test --net elastic -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node" {docker-image}
+docker run --name es-node01 -p 9200:9200 -p 9300:9300 -it {docker-image}
```

Comment:
let's leave the `--net elastic` , I think it's good practice to have a specific docker network